src/consumer/base_consumer.rs

```diff
@@ -145,7 +144,7 @@ impl<C: ConsumerContext> BaseConsumer<C> {
     /// #    .create()
     /// #    .unwrap();
     ///
```

Comment:
A minor cosmetic improvement: I noticed that these empty lines will show up in [the docs](https://fede1024.github.io/rust-rdkafka/rdkafka/consumer/base_consumer/struct.BaseConsumer.html#method.iter). I suggest removing them.